docs/plugins/internationalization.md

```diff
@@ -149,26 +149,53 @@ export const myPluginTranslationRef = createTranslationRef({
 
 ## For an application developer overwrite plugin messages
 
+Step 1: Create translation resources
+
+```ts
+// packages/app/src/translations/userSettings.ts
+
+import { createTranslationResource } from '@backstage/core-plugin-api/alpha';
+import { userSettingsTranslationRef } from '@backstage/plugin-user-settings/alpha';
+
+export const userSettingsMessages = createTranslationResource({
+  ref: userSettingsTranslationRef,
+  translations: {
+    zh: () =>
+      Promise.resolve({
+        default: {
+          'languageToggle.title': '语言',
+          'languageToggle.select': '选择{{language}}',
+          'languageToggle.description': '切换语言',
+          'themeToggle.title': '主题',
+          'themeToggle.description': '切换主题',
+          'themeToggle.select': '选择{{theme}}',
+          'themeToggle.selectAuto': '选择自动主题',
+          'themeToggle.names.auto': '自动',
+          'themeToggle.names.dark': '暗黑',
+          'themeToggle.names.light': '明亮',
+        },
+      }),
+  },
+});
+```
+
+You should change `zh` under the translations object to your local language.
+
+Step 2: Config translations in `packages/app/src/App.tsx`
+
 In an app you can both override the default messages, as well as register translations for additional languages:
 
 ```diff
++ import { userSettingsMessages } from './translations/userSettings';
+
  const app = createApp({
 +  __experimentalTranslations: {
 +    availableLanguages: ['en', 'zh'],
-+    resources: [
-+      createTranslationMessages({
```

Comment:
I think this wanted to actually show off `createTranslationMessages` too. But ok let's merge and iterate